instrumentation/jaxrs-client/jaxrs-client-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jaxrsclient/v1_1/ClientHandlerInstrumentation.java

```diff
@@ -39,7 +39,7 @@ public ElementMatcher<TypeDescription> typeMatcher() {
   public void transform(TypeTransformer transformer) {
     transformer.applyAdviceToMethod(
         named("handle")
-            .and(takesArgument(0, extendsClass(named("com.sun.jersey.api.client.ClientRequest"))))
+            .and(takesArgument(0, named("com.sun.jersey.api.client.ClientRequest")))
             .and(returns(extendsClass(named("com.sun.jersey.api.client.ClientResponse")))),
```

Comment:
supporting narrowing of return type makes some sense, but not for params since wouldn't be an override anymore

Reply:
even if the method has a more specific return type the caller will still call the bridge method with the original return type that will forward the call (unless the caller uses reflection) so we probably can remove the `extendsClass` return type too